### DISCUSSION TITLE
Error Installing astroid_template_two_quickstart on Joomla - Missing XML File

### DISCUSSION THREAD

**Discussion by ali13** [templaza/astroid-framework#746] [General]
Hello,

I am trying to install the astroid_template_two_quickstart on Joomla, but I am encountering an error. Joomla is returning a message indicating that the XML file is missing.

Could you please help me resolve this issue?

Thank you in advance for your assistance.

**Comment by sonvnn** [comment]
Quickstart is not an extension. You can not install it via Joomla install. 
To install quickstart. 
1. You have to upload it to your server. 
2. Extract package
3. Go to your url using browser and install it.

Best Regards,
Sonny

**Comment by Chacapamac** [comment]
TUTORIAL
https://www.akeeba.com/documentation/akeeba-solo/extracting-with-kickstart.html
